web-dapp/server-config.js

```diff
@@ -48,6 +47,11 @@ if (!process.env.REACT_APP_POPA_CONTRACT_ADDRESS) {
     throw new Error('REACT_APP_POPA_CONTRACT_ADDRESS env var is not defined');
 }
 
+// NOTE: this is in wei. If this value is set, other price_* options will be ignored,
+if (process.env.REACT_APP_PRICE_ETH) {
+    cfg.priceWei = web3.toBigNumber(process.env.REACT_APP_PRICE_ETH).mul(web3.toBigNumber('1e+18'));
+}
```

Comment:
What should we do if this env variable is not set? Throw an error or use a default?

Reply:
You're right, I think we should thrown an error, because dynamic prices are not implemented.